.pre-commit-config.yaml

```diff
@@ -0,0 +1,32 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks.git
+    sha: v0.9.5
+    hooks:
+    -   id: autopep8-wrapper
+    -   id: check-added-large-files
+    -   id: check-docstring-first
+    -   id: check-executables-have-shebangs
+    -   id: check-json
+    -   id: check-merge-conflict
+    -   id: check-xml
+    -   id: check-yaml
+    -   id: debug-statements
+    -   id: detect-private-key
+    -   id: double-quote-string-fixer
+    -   id: end-of-file-fixer
+    -   id: file-contents-sorter
+    -   id: flake8
+    -   id: mixed-line-ending
+    -   id: name-tests-test
+    -   id: requirements-txt-fixer
+    -   id: sort-simple-yaml
+    -   id: trailing-whitespace
+-   repo: https://github.com/asottile/reorder_python_imports.git
+    sha: v0.3.5
+    hooks:
+    -   id: reorder-python-imports
+-   repo: https://github.com/Lucas-C/pre-commit-hooks.git
+    sha: v1.1.1
+    hooks:
+    -   id: remove-tabs
+        files: \.py$
```

Comment:
`types: [python]` is better than targeting by extension, since it can also detect and target files with a python shebang which don't have the extension

Reply:
Fixed by using the newer ocfweb config file.